style.css

```diff
@@ -11,53 +17,48 @@ body {
   height: 100vh;
 }
 .left-col {
-  background-color: gray;
   width: 300px;
   display: flex;
   flex-flow: column;
   justify-content: space-between;
 }
 .nav-menu {
-  border: 1px solid black;
```

Comment:
why did you remove this background completely?

the task was to add some color theme into it not to remove the available boxes